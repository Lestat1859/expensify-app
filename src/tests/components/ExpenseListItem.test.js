import React from 'react';
import {shallow} from 'enzyme';
import expenses from '../fixtures/expenses'; // data de test
import ExpenseListItem from '../../components/ExpenseListItem';

test('should return a component with the expense',()=> {
  const wrapper = shallow(<ExpenseListItem  {...expenses[1] }/>); // on destructure l'objet pour correspondre a l'attendu du composant

  expect(wrapper).toMatchSnapshot();
});