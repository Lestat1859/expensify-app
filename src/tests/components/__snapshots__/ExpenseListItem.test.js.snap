// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return a component with the expense 1`] = `
<div>
  <Link
    replace={false}
    to="/edit/2"
  >
    <h3>
      Rent
    </h3>
  </Link>
  <p>
    109500
     - 
    -345600000
  </p>
</div>
`;
